store/LocalStore.js

```diff
@@ -29,7 +29,7 @@ LocalStore.prototype.incr = function (ip, cb) {
     cb(null, { current, ttl: this.timeWindow })
   } else {
     // start counting from the first request/increment
-    if (!this.msLastBeat) {
+    if (this.msLastBeat === undefined || this.msLastBeat === null) {
```

Comment:
The existing logical not operator already catches falsy values, mind clarifying on this change?

Reply:
Yes, of course

Sinon fake timers set the starting time to be 0 and it's false so...

Reply:
I know but:
1. this helps with debugging as we can see how much time passed
2. #217 remove this check anyway...

Reply:
Ok!